borg/locking.py

```diff
@@ -8,17 +8,22 @@
 ADD, REMOVE = 'add', 'remove'
 SHARED, EXCLUSIVE = 'shared', 'exclusive'
 
-# only determine the PID and hostname once.
-# for FUSE mounts, we fork a child process that needs to release
-# the lock made by the parent, so it needs to use the same PID for that.
-_pid = os.getpid()
+# for performance reasons, only determine the hostname once.
+## XXX this sometimes requires live internet access for issuing a DNS query in the background.
+##_hostname = '%s@%s' % (socket.getfqdn(), uuid.getnode())
 _hostname = socket.gethostname()
```

Comment:
gethostname should only look in uname(2) or equivalent, so it should never require Internet access. Keep it as is for 1.0.

(Which reminds me that there's a sentence in the 1.1 docs saying something like "Borg doesn't do networking" which could be considered incorrect in certain setups due to the getfqdn() call involving a resolver).